module/vision/GoalDetector/src/GoalDetector.cpp

```diff
@@ -240,16 +243,17 @@ namespace module::vision {
                              *                  THROWOUTS                  *
                              ***********************************************/
 
-                            if (config.debug) {
-                                log<NUClear::DEBUG>("**************************************************");
-                                log<NUClear::DEBUG>("*                    THROWOUTS                   *");
-                                log<NUClear::DEBUG>("**************************************************");
-                            }
+
                             bool keep = true;  // if false then we will not consider this as a valid goal post
 
                             // If the goal is too far away, get rid of it!
                             if (distance > config.max_goal_distance) {
                                 keep = false;
+                                if (config.debug) {
```

Comment:
Modernise debug logging.

Reply:
This should be handled in a separate PR. It was added 3 days ago in another PR by someone else.

Reply:
All of these files should have this changed in separate PRs